modules/standard/Regexp.chpl

```diff
@@ -138,19 +476,25 @@ record stringPart {
     Regular expression search routines normally return one of these.
     Also, this type can be passed as a capture group argument.
     Lastly, something of type reMatch can be checked for a match
```

Comment:
These lines seem intentionally separated in the comment, but end up in the same paragraph in the generated html.  Should they be split up there?